core/services/synchronization/stats_pusher.go

```diff
@@ -73,32 +80,73 @@ func (eq *StatsPusher) Close() error {
 	return eq.WSClient.Close()
 }
 
-func (eq *StatsPusher) pollEvents(parentCtx context.Context) {
+type response struct {
+	Status int `json:"status"`
+}
+
+func (eq *StatsPusher) syncEvent(event *models.SyncEvent) error {
+	eq.WSClient.Send([]byte(event.Body))
+
+	message, err := eq.WSClient.Receive()
+	if err != nil {
+		return err
+	}
+
+	var response response
+	err = json.Unmarshal(message, &response)
+	if err != nil {
+		return err
+	}
+
+	if response.Status != 201 {
```

Comment:
Should we make this `<300` for robustness? If we add a `200` in `explorer` it would be nice if this wasn't so fickle.

Reply:
I prefer `precise` to fickle, I don't think it's a good idea to support status codes we haven't even defined, this is not HTTP.